src/js/user-profile/actions/index.js

```diff
@@ -97,3 +97,71 @@ export function removeSavedForm(formId) {
       .catch(() => dispatch(removingSavedFormFailure()));
   };
 }
+
+export function fetchExtendedProfile() {
```

Comment:
FYI we will be breaking this out into multiple actions to "progressively enhance" the profile object based on the endpoints available, e.g. address, military history, etc.